lib/views/home/home_view.dart

```diff
@@ -158,6 +164,14 @@ class _HomeViewState extends State<HomeView> {
                 shrinkWrap: true,
                 itemCount: snapshot.data!.length,
                 itemBuilder: (context, index) {
+                  /// Returning a null widget when the invoice is unpaid
+                  if (snapshot.data![index].identifier == "invoice" &&
+                      snapshot.data![index].status == "unpaid") {
+                    return const SizedBox(
+                      width: 0,
+                      height: 0,
+                    );
+                  }
```

Comment:
this is the wrong place where to filter, you should add the code inside the `listInvoices` and pass the status `paid` to the method

We need to show the money that we had, so the unpaid invoice are not the right one, but the paid one are